### ISSUE TITLE
Does this support Python 3?

### ISSUE THREAD

**Issue by Flimm** [TriplePoint-Software/audit_trail#21]
`setup.py` does not contain classifiers indicating support for Python 3, and `.travis.yml` and `tox.ini` both don't test against Python 3.

Is this library meant to support Python 3?

**Comment by whitespy** [comment]
Yes, it is. Python 3 support has been added in commit https://github.com/TriplePoint-Software/django_audit_trail/commit/eb99d6ea942e48eb680fa33cb9b7f832486ed629

**Comment by syabro** [comment]
@Flimm willing add python3 tests soon, but it works with py3
